### PR TITLE
feat: genesis block injection for `Multiplanetary`

### PR DESCRIPTION
**Pull request by boscohyun**
- add genesis block for planet 0x000000000000(Odin).
- add genesis block for planet 0x000000000001(Heimdall).
- consider several genesis blocks in `BlockManager`.

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1205698971357142

### CODE REVIEW

File: nekoyume/Assets/_Scripts/Blockchain/BlockManager.cs [hunk 1]
```diff
@@ -84,32 +84,52 @@ public static Block ImportBlock(string path)
...
+                     uri.AbsolutePath.StartsWith("/planets/0x"))
+            {
+                localPath = Platform.GetStreamingAssetsPath(uri.LocalPath[1..]);
+            }
+            else
+            {
+                localPath = path;
+            }
+
+            Debug.Log($"BlockManager] localPath: {localPath}");
```
Comment (longfin) [inline nekoyume/Assets/_Scripts/Blockchain/BlockManager.cs:106]:
```suggestion
            Debug.Log($"[BlockManager] localPath: {localPath}");
```

File: nekoyume/Assets/_Scripts/Blockchain/BlockManager.cs [hunk 2]
```diff
@@ -84,32 +84,52 @@ public static Block ImportBlock(string path)
...
-                ? Platform.GetStreamingAssetsPath("genesis-block-9c-main")
-                : path;
+            var uri = new Uri(path);
+            // NOTE: Check cached genesis block in local path.
+            string localPath;
+            if (uri.IsFile)
+            {
+                localPath = path;
+            }
+            else if (uri.AbsolutePath.Equals("/genesis-block-9c-main") ||
```
Comment (longfin) [inline nekoyume/Assets/_Scripts/Blockchain/BlockManager.cs:96]: Could we apply generalized `planets/<id>` convention against `genesis-block-9c-main` too?

File: .gitignore [hunk 3]
```diff
@@ -87,7 +87,7 @@ nekoyume/compose/results
 # Genesis Block
 /nekoyume/Assets/StreamingAssets/genesis-block*
 /nekoyume/Assets/StreamingAssets/genesis-block-dev*
-!/nekoyume/Assets/StreamingAssets/genesis-block-9c-main
```
Comment (longfin) [inline .gitignore:90]: Is this still needed?

File: nekoyume/Assets/_Scripts/Blockchain/BlockManager.cs (hunk 2 above)
Comment (boscohyun) [reply nekoyume/Assets/_Scripts/Blockchain/BlockManager.cs:96]: The [planet registry](https://planets.nine-chronicles.com/planets/index.json) still returns "https://release.nine-chronicles.com/genesis-block-9c-main".
And it also covered "https://planets.nine-chronicles.com/planets/0x000000000000/genesis".

File: .gitignore (hunk 3 above)
Comment (boscohyun) [reply .gitignore:90]: Maybe?

Comment (boscohyun) [reply .gitignore:90]: Hm.. I'll change the genesis block path to /planets/0x000000000000/genesis if it is genesis-block-9c-main.

**Review by jonny-jeahyunchoi — APPROVED** [review]

**Review by sonohoshi — APPROVED** [review]

**Pull request merged**